exts/base.py

```diff
@@ -141,7 +141,8 @@ async def send_group_help(self, group):
         embed.title = await self.get_command_signature(group)
         embed.description = group.help
         for cmd in await self.filter_commands(group.commands, sort=True):
-            embed.add_field(name=await self.get_command_signature(cmd), value=cmd.help, inline=False)
+            embed.add_field(name=await self.get_command_signature(cmd), value=cmd.help if cmd.help else "",
```

Comment:
This seems odd, do they really no longer accept null values there?

Reply:
might have been for mypy